LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Dräger
+Copyright (c) 2020 Draeger
```

Comment:
```suggestion
Copyright (c) 2024 Draeger
```

The year of first publication of the current revision (, which is a derivative work,) is 2024.
So in my opinion at least the year 2024 should be included in case any date is given.
Additional years are fine, although unnecessary.

For example "2020, 2021, 2022, 2023, 2024" would be fine for me as well, although "2024" alone suits the template given in https://github.com/Draegerwerk/sdc11073/pull/377#discussion_r1721638127 better.

Thus the notice "Copyright (c) 2024 Draeger" seems to be the best solution.

Reply:
The work on this project started in 2020.
I believe, "Copyright (c) 2020 Draeger" is fine.